ngraph/src/ngraph/frontend/onnx_import/onnx.cpp

```diff
@@ -43,17 +43,50 @@ namespace ngraph
 
                 struct stream_parse : ngraph_error
                 {
-                    explicit stream_parse(std::istream&)
-                        : ngraph_error{"Failure parsing data from the provided input stream"}
+                    explicit stream_parse(const std::string& stream_type)
+                        : ngraph_error{"Failure parsing data from the provided input " +
+                                       stream_type + "."}
+                    {
+                    }
+                };
+
+                struct stream_corrupted : ngraph_error
+                {
+                    explicit stream_corrupted()
+                        : ngraph_error{"Provided input stream is in incorrect state."}
                     {
                     }
                 };
 
             } // namespace error
-        }     // namespace detail
+
+            std::shared_ptr<Function> get_ng_function(const ONNX_NAMESPACE::ModelProto& model_proto)
```

Comment:
Just a suggestion to consider - how about renaming it to `to_ng_function`?

Reply:
What about `convert_to_ng_function`?

Reply:
That will work too although I personally prefer shorter names.

Reply:
Me too ;)